sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -1814,7 +1814,7 @@ private DataFeed GetDataFeedWithOptionalMembersSet(string name, DataFeedSource d
             {
                 Name = name,
                 DataSource = dataSource,
-                Granularity = new DataFeedGranularity(DataFeedGranularityType.Custom) { CustomGranularityValue = 1360 },
+                Granularity = new DataFeedGranularity(DataFeedGranularityType.Custom) { CustomGranularityValue = 3000 },
```

Comment:
The service added some extra validation when creating a data feed, and was complaining that our granularity value was too low. Increased it and needed to rerecord a lot of tests.

Reply:
I imagine this is something the user knows that needs to adjust? is 300 an arbitrary number? or a recommended value from service

Reply:
Completely arbitrary. They need to improve a lot of their error messages.

Reply:
I'll create an issue for this message specifically.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/22557